File: Javascript/SimplePrompt.js
// cd to /GPT-3_API/Javascript
// run node SimplePrompt.js
const axios = require('axios');

const fs = require('fs');
fs.readFile('../API_KEY.txt', 'utf8', function(err, apiKey) {
    if (err) throw err;

    const client = axios.create({
        headers: { 'Authorization': 'Bearer ' + apiKey }
    });

    const data = {
        'prompt': 'English: Hello my name is Simon and i am a student for ABB working in tech and i will do a presentation about GPT-3\nSwedish:',
        'max_tokens': 50,
        'top_p': 1,
        'temperature': 1,
        //'stop': ['English']
    }

    client.post('https://api.openai.com/v1/engines/davinci/completions', data)
    .then(result => {
        console.log(data.prompt + result.data.choices[0].text);
    }).catch(err => {
        console.log(err);
    });
});

File: Python/SimplePrompt.py
import requests                                 # library used to call the api
import json                                     # library used to handle json formats

apiKey = open("API_KEY.txt").read()             # get secret API key

headers = {                                     # used to pass data with request
    "Content-Type": "application/json",         # specifies sending datatype
    "Authorization": "Bearer " + apiKey         # holds authorization information
}

data = json.dumps({
    "prompt": "Somebody once told me the world is gonna roll me",   # text gpt-3 will continue on
    "max_tokens": 15                            # specifies gpt-3's response length
})

result = requests.post(
    "https://api.openai.com/v1/engines/davinci/completions",    # endpoint
    headers=headers,
    data=data
)

print(json.loads(data)["prompt"] + result.json()["choices"][0]["text"]) # print formated result